examples/EUVAC/json-schemas/vaccinationDisclosure.BorderControl.json

```diff
@@ -1,53 +1,70 @@
 ﻿{
-    "$schema" : "http://json-schema.org/schema#",
-    "$id" : "https://github.com/nl-eHealth-experimental/examples/EUVAC/vaccinationDisclosure.BorderControl.schema.json",
-    "type" : "object",
-    "properties" : {
-      "dsc": { "type": "string", "minLength": 12, "maxLength": 12, "$comment": "8 bytes of SHA256 of DSC certificate in base45." },
-      "nam": { "type": "string", "$comment": "legal name" },
-      "v": { "type": "array", "items": "#/definitions/vaccinationEvent", "minItems": 1 },
-      "c": { "type": "#/definitions/certificate" },
-      "t": { "type": "#/definitions/test" },
-      "r": { "type": "#/definitions/recovery" }
-    },
-    "required": ["dsc","nam","v","c"],
-    "definitions" : {
-        "vaccinationEvent" : {
-            "type" : "object",
-            "properties" : {
-                "tg" : {"type":"string", "$comment" : "Disease or Agent Targeted" },
-                "fr" : {"type":"string", "format":"date", "$comment" : "Valid From" }
-            },
-            "required": ["tg","fr"]
-        },
-        "certificate": {
-            "type": "object",
-            "properties": {
-                "id" : {"type":"string", "$comment" : "Identifier UVCI" },
-                "is" : {"type":"string", "$comment" : "Issuer" },
-                "ia" : {"type":"string", "$comment" : "Issuing Authority" },
-                "st" : {"type":"string", "format":"date-time", "$comment" : "Valid from" },
-                "en" : {"type":"string", "format":"date-time", "$comment" : "Valid until" },
-                "vr" : {"type":"string", "$comment" : "Schema version, semver" }
-            },
-            "required": ["id", "is", "ia", "st", "en", "vr"]
-        },
-        "test": {
+  "$schema" : "http://json-schema.org/schema#",
+  "$id" : "https://github.com/nl-eHealth-experimental/examples/EUVAC/vaccinationDisclosure.Medical.schema.json",
+  "type" : "object",
+  "properties" : {
+    "nam": { "type": "string", "$comment": "Initial. FamilyName" },
+    "pid": { "type": "string", "$comment": "patient identifier e.g. NL BSN, UK NH ID." },
+    "sex": { "type": "string", "$comment": "patient Gender." },
+    "dob": { "type": "string", "format":"date", "$comment" : "Date of birth, ISO8601." },
+    "v": { "type": "array", "items": "#/definitions/vaccinationEvent", "minItems": 1 },
+    "t": { "type": "#/definitions/test" },
+    "r": { "type": "#/definitions/recovery" },
+    "c": { "type": "#/definitions/certificate" },
+    "dsc": { "type": "string", "minLength": 12, "maxLength": 12, "$comment": "8 bytes of SHA256 of DSC certificate in base45." }
+  },
+  "required": [ "nam", "pid", "dob", "sex", "v", "c", "dsc" ],
```

Comment:
sex only required for medical

Reply:
Aha! Its on all 3 on the google doc which is also now aligned.